web/pdf_page_view.js

```diff
@@ -260,6 +260,10 @@ class PDFPageView {
    */
   updatePosition() {
     if (this.textLayer) {
+      let readableStream = this.pdfPage.streamTextContent({
+        normalizeWhitespace: true,
+      });
+      this.textLayer.setTextContentStream(readableStream);
```

Comment:
We do, and I think it has something to do with the find code; at least [PDFFindController.updatePage]https://github.com/mozilla/pdf.js/blob/82f3145a5d8630378f378ef8f4f8596204273a6b/web/pdf_find_controller.js#L280 seems to me that it could be connected!?

We might want to try and make sense of the find code, and refactor that first?